app/styles/main.less

```diff
@@ -1,85 +1,7 @@
+// PatternFly
 @pf-less-path: "../../bower_components/patternfly/less";
-
```

Comment:
Nice!  Though I suppose we could ask if we need everything... guessing we probably do so this is cleaner.